client/signup/config/flows.js

```diff
@@ -269,8 +270,29 @@ function filterFlowName( flowName ) {
 	return flowName;
 }
 
+function filterDestination( destination, dependencies, flowName ) {
+	if ( abtest( 'guidedTours' ) === 'guided' ) {
+		const disabledForFlows = [ 'account', 'site-user' ];
+
+		if ( includes( disabledForFlows, flowName ) ) {
+			return destination;
+		}
+
+		const tourName = 'main';
+		//TODO: Build the query arg properly, so we don't have problems with
+		// destinations that already have query strings in
```

Comment:
Maybe sign this TODO, so we can get pinged if we ever forget? (Also, spacing before "TODO", and place it right before the return statement rather than after the `tourName` declaration).